oktoberfest/predict/dlomix.py

```diff
@@ -0,0 +1,163 @@
+import contextlib
+import logging
+import multiprocessing as mp
+import os
+import sys
+import warnings
+from pathlib import Path
+from typing import Dict, Union
+
+import numpy as np
+from spectrum_fundamentals.constants import ALPHABET, ANNOTATION
+from spectrum_fundamentals.mod_string import parse_modstrings
+from spectrum_io.file.parquet import write_file
+
+sys.path.append("/cmnfs/home/students/j.schlensok/dlomix/bmpc_shared_scripts/oktoberfest_interface")
```

Comment:
please remove hardcoded statement. Will this be a requirement later on or is it just for now?